doc/user/content/sql/system-catalog.md

```diff
@@ -249,6 +249,19 @@ Field          | Type       | Meaning
 `schema_id`    | [`bigint`] | The ID of the schema to which the sink belongs.
 `name`         | [`text`]   | The name of the sink.
 
+### `mz_source_info`
+
+The `mz_source_info` table contains a row for each source in the system.
```

Comment:
for each partition of each source, yeah?

Reply:
Good catch! Fixed!